File: Aula01_Ex03_area_retangulo.py
base = float(input("Insira aqui a base do triângulo: "))
altura = float(input("Agora, insira aqui a altura do triângulo: "))

area = base * altura
perimetro = (base*2) + (altura*2)

print("A área do triângulo é {:.2f} e o perímetro dele é {:.2f}!".format(area, perimetro))

File: Aula01_Ex03_area_retangulo.R
base<-as.numeric(readline(prompt="Insira o valor da base do retângulo: "))
altura<-as.numeric(readline(prompt="Insira o valor da altura do retângulo: "))

area<-base*altura
perimetro<-(base*2)+(altura*2)

print(paste("a área do retângulo é", area, "e o perímetro é", perimetro))
